### ISSUE TITLE
Plugin overrides are not taken into account with scheduler

### ISSUE THREAD

**Issue by arkhi** [getgrav/grav-plugin-email#122]
Hello,

- I have a local development at http://my-website.com.local/.
- I have the following structure in my user folder:
    ```
    user/
      |- config/
        |- plugins/
          |- … (no email.yaml)
      |-…
      |- my-website.com.local/
        |- config/
          |- plugins/
            |- email.yaml
    ```
- Enabling the scheduler and saving the settings in the plugin configuration page (/admin/plugins/email), they are saved in _user/my-website.com.local/config/plugins/email.yaml_ as expected.
- With this, the scheduler will not be enabled.
- Copying the same configuration in _user/config/plugins/email.yaml_, the scheduler sends the email as planned.

Based on the [plugin overrides section of Grav](https://learn.getgrav.org/16/advanced/environment-config#plugin-overrides), I think there is an issue. I checked a few times but I can’t see a problem on my side.

The only hiccup I have is the scheduler keeps on telling “ Cron Not Available” in the admin (/admin/tools/scheduler), although a `bin/grav scheduler -i` gives me:

```
Install Scheduler
=================
                                                                                                                        
 [OK] All Ready! You have already set up Grav's Scheduler in your crontab                                                                                                                                     

 ! [NOTE] To install, run the following command from your terminal:                                                     

 (crontab -l; echo "* * * * * cd /vagrant;/usr/bin/php7.2 bin/grav scheduler 1>> /dev/null 2>&1") | crontab -
```

If anyone can give it a try and confirm (and fix :) ) or infirm, that’d be great, thanks!

**Comment by arkhi** [comment]
One more detail: The environment is running within a vagrant box. Don’t know if that should make a difference from Grav’s point of view.

**Comment by mahagr** [comment]
Yes, the configuration is saved locally to the site you're using. Currently, there is no way to modify the main configuration unless you're running the site using it.

**Comment by arkhi** [comment]
Shouldn’t the local configuration be taken into account when running the local version if no global configuration exists?
